sourcing-new/src/main/scala/ch/epfl/bluebrain/nexus/sourcingnew/projections/ProjectionId.scala

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.bluebrain.nexus.sourcingnew.projections
+
+sealed trait ProjectionId {
+  def value: String
+}
+
+final case class CacheProjectionId(value: String) extends ProjectionId
+
+final case class ViewProjectionId(value: String) extends ProjectionId
+
+final case class SourceProjectionId(value: String) extends ProjectionId
+
+final case class CompositeViewId(sourceId: String, projectionId: String) extends ProjectionId {
```

Comment:
If for normal view is called `ViewProjectionId` shouldn't this be called `CompositeViewProjectionId`?